src/legacy/core_plugins/metrics/public/_variables.scss

```diff
@@ -1,11 +1,11 @@
-$tvbLineColor: transparentize($euiColorFullShade,0.8);
-$tvbLineColorReversed: transparentize($euiColorEmptyShade,0.6);
+$tvbLineColor: transparentize($euiColorFullShade, .8);
```

Comment:
For the record, I personally prefer the `0.` prefix, because it's easy to mistake numbers without it. I think we have a JS lint rule that enforces the leading `0.`, but this is just a nit pick. Not a required change.

Reply:
Yeah as a design team, we've decided to keep the leading 0's off and it's part of the linter we use in here and EUI.